src/contexts/PermissionsContext.js

```diff
@@ -13,9 +14,17 @@ import { getEmptyAddress } from '../web3-utils'
 const { Provider, Consumer } = React.createContext()
 
 class PermissionsProvider extends React.Component {
+  static propTypes = {
+    apps: PropTypes.array.isRequired,
+    children: PropTypes.node.isRequired,
+    permissions: PropTypes.oneOfType([PropTypes.array, PropTypes.object])
```

Comment:
Hmm... this definitely seems a bit odd... `permissions` should only ever be an array.

Reply:
![image](https://user-images.githubusercontent.com/3072458/48276821-3eabeb80-e449-11e8-9e46-babb48f6b407.png)

What do you think? set it back to `object` too, to avoid the error?

Reply:
In the case where it is an object I am seeing a Proxy object

Reply:
@sohkai are you certain `permissions` should only be an array?
I am seeing it as an `object`, with a shape of:

```js
hashN: {
  hashN: 
    allowedEntities: array // (of hashes)
    manager: hash
}
```

And when I set the default value to object and only expect object there is no warning/error.

Reply:
Ahh, nice catch 👍!